pkg/crc/config/viper_config.go

```diff
@@ -187,12 +187,12 @@ func runCallbacks(callbacks []SetFn, key interface{}, value interface{}) string
 func Set(key string, value interface{}) (string, error) {
 	_, ok := allSettings[key]
 	if !ok {
-		return "", fmt.Errorf("Config property '%s' does not exist", key)
+		return "", fmt.Errorf("crc does not understand configuration property '%s'", key)
```

Comment:
no need to include `crc` here. This is not an old Sierra parser... ;-)

`Configuration property %s does not exist` is good for me

Reply:
+1